devices/button.py

```diff
@@ -0,0 +1,106 @@
+import threading
+import time
+import pyudev
+from opcua import ua
+
+from .base import (
+    BaseDevice,
+    DeviceSubscriptionHandler,
+    logger,
+    COUNT_REGISTER,
+    STATE_REGISTER,
+    TIME_REGISTER,
+    TYPE_REGISTER,
+    VIRTUALIZED_REGISTER,
+)
+
+
+class Button(BaseDevice):
+    def __init__(self, name="VirtualButton", pin=None, update_interval=0.1):
```

Comment:
```suggestion
class VirtualButton(BaseDevice):
    def __init__(self, name="VirtualButton", pin=None, update_interval=0.1):
```

I'll match the name, but if there are multiple devices allowed, I wouldn't set a default name, unless it's a default prefix name and you add the number of device next to it